### DISCUSSION TITLE
Error on Sync with Windows

### DISCUSSION THREAD

**Discussion by rjkantor** [TriliumNext/Trilium#2552] [General]
Hello,

I have setup my Mac Desktop and a Ubuntu Server for Trilium.  I have synced my Mac to the Server successfully.
When I try to sync the Ubuntu Server to my Win Desktop, I keep getting a redirect error.  Any ideas?
I have pasted the error log from the Win Trilium.

10:47:54.870 DB not initialized, please visit setup page
10:47:54.872 DB size: 4 KB
10:47:54.994 Generated session secret
10:47:56.005 App HTTP server starting up at port 37840
10:47:56.006 {
  "appVersion": "0.49.4",
  "dbVersion": 189,
  "syncVersion": 23,
  "buildDate": "2022-01-09T22:32:13+01:00",
  "buildRevision": "be59f248e8387c268471946da99979939c8d2c1d",
  "dataDirectory": "C:\\Users\\xxxx\\AppData\\Roaming\\trilium-data",
  "clipperProtocolVersion": "1.0"
}
10:47:56.006 CPU model: Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, logical cores: 8 freq: 1992 Mhz
10:47:56.020 Listening on port 37840
10:48:22.244 Getting document options FROM sync server.
10:48:22.566 Creating database for sync
10:48:22.582 Schema and not synced options generated.
10:48:22.582 Triggering sync.
10:48:22.584 Slow 200 POST /api/setup/sync-from-server with 20 bytes took 340ms
10:48:22.597 sync failed: Request to GET https://trilium.xxxxx.com/api/setup/status failed, error: Error: net::ERR_TOO_MANY_REDIRECTS
stack: Error: Request to GET https://trilium.xxxx.com/api/setup/status failed, error: Error: net::ERR_TOO_MANY_REDIRECTS
    at generateError (C:\Users\xxxxx\Downloads\trilium-windows-x64-0.49.4\resources\app.asar\src\services\request.js:191:12)
    at ClientRequest.<anonymous> (C:\Users\xxx\Downloads\trilium-windows-x64-0.49.4\resources\app.asar\src\services\request.js:58:47)
    at ClientRequest.emit (node:events:394:28)
    at ClientRequest._die (node:electron/js2c/browser_init:101:8391)
    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:101:7128)
    at SimpleURLLoaderWrapper.emit (node:events:394:28)
    at SimpleURLLoaderWrapper.callbackTrampoline (node:internal/async_hooks:130:17)
10:48:23.598 Returning sync stats: {"initialized":false,"outstandingPullCount":0}
10:48:23.598 304 GET /api/sync/stats with 46 bytes took 1ms
10:48:24.591 Returning sync stats: {"initialized":false,"outstandingPullCount":0}
10:48:24.591 304 GET /api/sync/stats with 46 bytes took 0ms
10:48:25.592 Returning sync stats: {"initialized":false,"outstandingPullCount":0}
10:48:25.593 304 GET /api/sync/stats with 46 bytes took 1ms
10:48:26.591 Returning sync stats: {"initialized":false,"outstandingPullCount":0}
10:48:26.591 304 GET /api/sync/stats with 46 bytes took 0ms
10:48:27.591 Returning sync stats: {"initialized":false,"outstandingPullCount":0}
10:48:27.591 304 GET /api/sync/stats with 46 bytes took 0ms
10:48:28.591 Returning sync stats: {"initialized":false,"outstandingPullCount":0}
10:48:28.592 304 GET /api/sync/stats with 46 bytes took 1ms

**Comment by zadam** [comment]
Hello, Trilium on this endpoint (sync/status) does not issue any redirects (it's a REST API), so I guess this is some proxy server between the client and server doing this.
